src/main/java/xbot/common/command/BaseRobot.java

```diff
@@ -174,6 +174,7 @@ protected void initializeSystems() {
         // Get the property manager and get all properties from the robot disk
         propertyManager = injectorComponent.propertyManager();
         xScheduler = injectorComponent.scheduler();
+        xScheduler.reset();
```

Comment:
⭐ consider a comment here about why this is important (and that it's mainly for tests?)